code_share/code_share/views.py

```diff
@@ -0,0 +1,17 @@
+from django.shortcuts import render
+
+
+def page_not_found_view(request):
```

Comment:
Just use the HTTPResponse error classes in the django.http.response.py,You dont need to render them, they are automatically rendered by django at production time(DEBUG=False).